src/LiveComponent/doc/index.rst

```diff
@@ -791,6 +791,22 @@ Want to change the value of a model or even trigger an action from your
 own custom JavaScript? No problem, thanks to a JavaScript ``Component``
 object, which is attached to each root component element.
 
+Before going any further, check that the controller entry ``@symfony/ux-live-component`` has been added to ``assets/controllers.json``, like this::
```

Comment:
I think I know what goes wrong here. You installed the LiveComponent bundle **before** the StimulusBundle. This is explained in the doc here https://symfony.com/bundles/ux-live-component/current/index.html#installation : 

>  Before you start, make sure you have [StimulusBundle configured in your app]

But like you, I made the mistake multiple times.
So I don't think we should talk about the assets/controller.json here, but maybe we can add a more detailed section on how to install the Stimulus Bundle directly into this page. Or maybe write something more visual that the StimulusBundle should be written first. And I wonder if when installing the LiveComponentBundle we can execute the StimulusBundle recipe?